vue/components/ui/atoms/container/container.vue

```diff
@@ -72,6 +106,11 @@ export const Container = {
             if (this.mode) base[`container-ripe-${this.mode}`] = true;
             return base;
         }
+    },
+    methods: {
+        onButtonIconClick(event, buttonEvent) {

```

Comment:
Please use the reverse order to match the rest of the codebase: the native event first. Also, rename `event` to `buttonId` or something, in order to avoid confusion .